package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/node-jspm-jasmine.js",
   "scripts": {
     "test": "test",
-    "build": "babel src --out-dir lib",
+    "build": "./node_modules/babel-cli/bin/babel.js src --out-dir lib",
```

Comment:
curious why you made this change, since npm [adds executables to the PATH when executing scripts](https://docs.npmjs.com/misc/scripts#path)?